scripts/forward-bash/forward.sh

```
@@ -0,0 +1,284 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+# Colors and formatting
+BLUE='\033[0;34m'
+GREEN='\033[0;32m'
+YELLOW='\033[1;33m'
+RED='\033[0;31m'
+CYAN='\033[0;36m'
+DIM='\033[2m'
```

Shellcheck

[warning] DIM appears unused. Verify use (or export if used externally).

(SC2034)